### PR TITLE
change CCPi CIL version

### PR DESCRIPTION
**Pull request by paskino**
updates the CIL tag to latest 20.04 release

### CODE REVIEW

File: SuperBuild/External_CCPi-Regularisation-Toolkit.cmake [hunk 1]
```diff
@@ -81,6 +81,7 @@ if(NOT ( DEFINED "USE_SYSTEM_${externalProjName}" AND "${USE_SYSTEM_${externalPr
       # external projects
       CONFIGURE_COMMAND ${CMAKE_COMMAND} -E env CIL_VERSION=${${proj}_TAG} ${CMAKE_COMMAND} ${${proj}_SOURCE_DIR}
         -DCMAKE_INSTALL_PREFIX:PATH=${${proj}_INSTALL_DIR}
+        -DCMAKE_BUILD_TYPE:STRING=Release
```
Comment (KrisThielemans) [inline SuperBuild/External_CCPi-Regularisation-Toolkit.cmake:84]: I'm guessing this should be
```suggestion
        -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
```
@rijobro, what do you think?

**Comment by paskino** [comment]
closes #389

**Comment by KrisThielemans** [comment]
All green, except the final 4 docker tests whose CCPi tests still [fail with](https://travis-ci.org/github/SyneRBI/SIRF-SuperBuild/jobs/685868301#L165791)
```
OSError: libcilacc.so: cannot open shared object file: No such file or directory
```
However, it appears that these are still being build with the [previous version of the CIL libraries](https://travis-ci.org/github/SyneRBI/SIRF-SuperBuild/jobs/685868301#L2771)
```
-- CCPi-Regularisation-Toolkit URL https://github.com/vais-ral/CCPi-Regularisation-Toolkit.git
-- CCPi-Regularisation-Toolkit TAG db048a42018857c7aa30033e978a3600cbedb85f
```
@casperdcl, any clue why that would be?

I'll merge this anyway as it's not related to this PR

File: SuperBuild/External_CCPi-Regularisation-Toolkit.cmake (hunk 1 above)
Comment (paskino) [reply SuperBuild/External_CCPi-Regularisation-Toolkit.cmake]: That's interesting, yesterday evening I didn't see this comment, but well I wasn't paying too much attention.

**Comment by casperdcl** [comment]
Yes those are `master`/devel builds so shouldn't be expected to pass pre-merge

**Pull request merged**